include.js

```diff
@@ -292,6 +300,13 @@ let cards = [
     author: 'Simar',
     githubLink: 'https://github.com/SimardeepSingh-zsh'
   },
+  {
+    artName: 'solarsystem',
+    pageLink: './Art/Joy/solarsystem.html',
```

Comment:
This should be `./Art/solasystem/solarsystem.html`

Reply:
@sophiabrandt done please check!